Assets/Scripts/Scenes/InGame/Player/TempPlayerMove.cs

```diff
@@ -105,7 +106,7 @@ void Start()
             //Shift+�ړ��L�[���������Ƃ��_�b�V����Ԃɐ؂�ւ�
             this.UpdateAsObservable()
                 .Where(_ => ((Input.GetKeyDown(dash) && Input.GetKey(KeyCode.W)) || (Input.GetKey(dash) && Input.GetKeyDown(KeyCode.W))) && !_isTiredPenalty && _moveVelocity != Vector3.zero)
-                .Where(_ => _isCanMove)
+                .Where(_ => _isCanMove && _isCannotMoveByParalyze == false)

```

Comment:
```suggestion
  .Where(_ => _isCanMove && !_isCannotMoveByParalyze)
``` 
隣の_isCanMoveが==trueを省略しているので、_isCannotMoveByParalyzeも省略して書くのが見やすいかもしれません。
他の_isCannotMoveByParalyze == falseも同様です。

Reply:
今日中に修正しておきます